### DISCUSSION TITLE
How to Set a Minimum Y-Axis Value in LineChart?

### DISCUSSION THREAD

**Discussion by ArturV93** [Abhinandan-Kushwaha/react-native-gifted-charts#872] [Q&A] [answered]
I'm working with the LineChart component, and I need the Y-axis to start from a value greater than zero, specifically 60.

Currently, I’ve managed to get this somewhat working using custom yAxisLabelTexts and adjusting the stepHeight. However, I'm unsure how to properly calculate the stepHeight based on the maxValue and noOfSections.

Is there a recommended way to handle this, or is there a simpler method to set a custom minimum value for the Y-axis (e.g., minYValue={60})?

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
@ArturV93 try the `yAxisOffset` prop
